services/core-api/app/api/incidents/models/mine_incident.py

```diff
@@ -74,33 +75,33 @@ class MineIncident(AuditMixin, Base):
 
     determination_type = db.relationship(
         'MineIncidentDeterminationType',
-        backref='mine_incidents',
+        backref='mine_incident',
         lazy='joined',
         uselist=False,
         foreign_keys=[determination_type_code])
     dangerous_occurrence_subparagraphs = db.relationship(
         'ComplianceArticle',
-        backref='mine_incidents',
+        backref='mine_incident',
         lazy='joined',
         secondary='mine_incident_do_subparagraph')
     followup_investigation_type = db.relationship(
         'MineIncidentFollowupInvestigationType',
-        backref='mine_incidents',
+        backref='mine_incident',
         lazy='joined',
         uselist=False)
-    mine_incident_category_code = db.Column(
-        db.String, db.ForeignKey('mine_incident_category.mine_incident_category_code'))
-    mine_incident_category = db.relationship('MineIncidentCategory', lazy='joined')
 
     recommendations = db.relationship(
         'MineIncidentRecommendation',
         primaryjoin=
         "and_(MineIncidentRecommendation.mine_incident_id == MineIncident.mine_incident_id, MineIncidentRecommendation.deleted_ind==False)",
-        lazy='selectin')
+        lazy='select')
```

Comment:
And this was a typo as well

Reply:
these are enumerated options. see here, selectin was used deliberately i believe. 

https://docs.sqlalchemy.org/en/13/orm/loading_relationships.html

Reply:
Good catch, I didn't see that in the docs.